src/main/ts/component/Editor.ts

```diff
@@ -105,17 +109,21 @@ class TinyMceEditor extends HTMLElement {
     });
   };
 
-  private _formDataHandler = (evt: Event) => {
+  private _formDataHandler = (evt: FormDataEvent): void => {
     const name = this.name;
-    if (name !== null) {
-      const data = (evt as any).formData as FormData;
-      data.append(name, this.value);
+    if (name != null) {
```

Comment:
Did you mean `!==` here?

Reply:
It was deliberate. We have a lint to check for `!=` for anything that is not null.

In this case while I don't expect `undefined` I did want to handle it.

Also I'm pretty sure you work somewhere else now! :-P

Reply:
:)

Reply:
There was a strange in-equality in the force that rippled through the fabric of git, that drew him to this PR

Reply:
I discovered the other day that my github account was using my Tiny email. After fixing that I've been getting lots of notifications :)

I only commented to say hi :)